CONTRIBUTING.md

```diff
@@ -66,7 +66,75 @@ The `metadata.json` file contains metadata of the module including `name`, `desc
 
 ```
 
-The `main.bicep` file is the public interface of the module. When authoring `main.bicep`, make sure to provide a description for each parameter and output. You are free to create other Bicep files inside the module folder and reference them as local modules in `main.bicep` if needed. You may also reference other registry modules to help build your module. If you do so, make sure to add them as external references with specific version numbers. You should not reference other registry modules through local file path, since they may get updated overtime.
+#### `main.bicep`
+
+The `main.bicep` file is the public interface of the module. When authoring `main.bicep`, make sure to provide a description for each parameter and output. You are free to create other Bicep files inside the module folder and reference them as local modules in `main.bicep` if needed. You may also reference other registry modules to help build your module. If you do so, make sure to add them as external references with specific version numbers. You should not reference other registry modules through local file path, since they may get updated overtime. The `main.bicep` most follow various static code analysis checks, such as including descriptions on every parameter and output. Default resource names should you abbirvations defined by the Azure [documentation](https://learn.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-best-practices/resource-abbreviations). The location parameter is required, and does not include a default value, to prevent Azure Marketplace Static Analysis issues when the module is being consumed.
```

Comment:
`abbirvations` -> typo?

Reply:
```suggestion
The `main.bicep` file is the public interface of the module. When authoring `main.bicep`, make sure to provide a description for each parameter and output. You are free to create other Bicep files inside the module folder and reference them as local modules in `main.bicep` if needed. You may also reference other registry modules to help build your module. If you do so, make sure to add them as external references with specific version numbers. You should not reference other registry modules through local file path, since they may get updated overtime. The `main.bicep` most follow various static code analysis checks, such as including descriptions on every parameter and output. Default resource names should you abbreviations defined by the Azure [documentation](https://learn.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-best-practices/resource-abbreviations). The location parameter is required, and does not include a default value, to prevent Azure Marketplace Static Analysis issues when the module is being consumed.
```

Reply:
`Default resource names should you abbreviations defined by the Azure`

you => use?